### DISCUSSION TITLE
Is elastalert highly available for alerts execution?

### DISCUSSION THREAD

**Discussion by akshay0709** [jertel/elastalert2#865] [Q&A] [answered]
Hey Folks,

Thanks for maintaining this project. I am evaluating this system for our use case. Wanted to check if this system has high availability while executing alerts? Most of you might be already using it and wanted to know how is your experience with adding multiple rules across multiple clusters?

**Comment by jertel** [comment, marked as answer]
Each ElastAlert 2 instance is independent. Running multiple instances with the same ruleset, against the same Elasticsearch cluster, will often result in duplicate alerts.

See https://github.com/jertel/elastalert2/discussions/544 for related info.
